src/coreclr/src/vm/methodtablebuilder.cpp

```diff
@@ -10461,6 +10461,13 @@ MethodTableBuilder::SetupMethodTable2(
         {
             pInstDest[j] = inst[j];
         }
+
+        if (pClass->GetDictionaryLayout() != NULL && pClass->GetDictionaryLayout()->GetMaxSlots() > 0)
```

Comment:
Do we ever have non-null dictionary layout with MaxSlots == 0?

Reply:
I remember hit such cases before when working on the original PR, but can't remember specifics at this point. I'll have to debug again to remember